plugins/module_utils/postgres.py

```diff
@@ -177,6 +177,9 @@ def get_conn_params(module, params_dict, warn_db_default=True):
     Kwargs:
         warn_db_default (bool) -- warn that the default DB is used (default True)
     """
+
+    ensure_required_libs(module)
```

Comment:
```suggestion
    ensure_required_libs(module)
```
I personally incline to calling `ensure_required_libs` in every module just before `get_conn_params`. It would require more work but, imo, the logic would be more clear - it would once be called at the beginning to ensure the lib is present.
If you prefer this way of solving the issue, i think we could remove `ensure_required_libs` from `connect_to_db` not to call it twice.